src/PrestaShopBundle/Controller/Admin/Improve/International/ZoneController.php

```diff
@@ -180,7 +180,7 @@ public function editAction(int $zoneId, Request $request): Response
             }
         }
 
-        if (!isset($editableZone) || !isset($zoneForm)) {
+        if (!isset($zoneForm)) {
```

Comment:
Why? :thinking:

Reply:
phpstan detected it, `$editableZone` cannot not be set (if that make sense ^^)